lib/settings/pages/theme_settings_page.dart

```diff
@@ -364,6 +400,250 @@ class _ThemeSettingsPageState extends State<ThemeSettingsPage> {
                       ],
                     ),
                   ),
+                  Container(
```

Comment:
It might be good to move this to a separate widget since it's quite large

Reply:
Done! This should be good to go now.